### ISSUE TITLE
Error when running yarn all:ovm

### ISSUE THREAD

**Issue by fainashalts** [ethereum-optimism/Truffle-ERC20-Example#7]
Hi there! I get an error when following the steps in your readme. Here is the output after running `yarn all:ovm`: 

```
  Contract: ERC20
Error: Invalid Signature
Error: Invalid Signature
    1) "before each" hook for "creation: should create an initial balance of 10000 for the creator"


  0 passing (5s)
  1 failing

  1) Contract: ERC20
       "before each" hook for "creation: should create an initial balance of 10000 for the creator":
     Uncaught Error: the string "abort(Error: Invalid Signature). Build with -s ASSERTIONS=1 for more info." was thrown, throw an Error :)
      at process.emit (node_modules/truffle/build/webpack:/node_modules/source-map-support/source-map-support.js:495:1)
      at process.emit (node_modules/truffle/build/webpack:/node_modules/source-map-support/source-map-support.js:495:1)
      at process._fatalException (internal/bootstrap/node.js:497:27)



error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Thanks, please let me know if you need any additional information!

**Comment by maxmalong** [comment]
I got the same problem

yarn run v1.22.10
$ yarn build:ovm && yarn test:ovm
$ truffle compile --config truffle-config-ovm.js
> Warning: possible unsupported (undocumented in help) command line option: --config
getting normal provider
got normal provider
we wrapping now??

Compiling your contracts...
===========================
we creating VM now??
> Compiling ./contracts/ERC20.sol
> Compiling ./contracts/IERC20.sol
> Compilation warnings encountered:

    /root/contract/Truffle-ERC20-Example/contracts/ERC20.sol:31:14: Warning: OVM: Taking arguments in constructor may result in unsafe code.
    ) public {
             ^

> Artifacts written to /root/contract/Truffle-ERC20-Example/build/contracts/ovm
> Compiled successfully using:
   - solc: 0.5.16

$ truffle test ./truffle-tests/test-erc20.js --config truffle-config-ovm.js
> Warning: possible unsupported (undocumented in help) command line option: --config
getting normal provider
got normal provider
we wrapping now??
we creating VM now??
Using network 'test'.


Compiling your contracts...
===========================
> Everything is up to date, there is nothing to compile.



  Contract: ERC20
Error: Invalid Signature
Error: Invalid Signature
    1) "before each" hook for "creation: should create an initial balance of 10000 for the creator"


  0 passing (3s)
  1 failing

  1) Contract: ERC20
       "before each" hook for "creation: should create an initial balance of 10000 for the creator":
     Uncaught Error: the string "abort(Error: Invalid Signature). Build with -s ASSERTIONS=1 for more info." was thrown, throw an Error :)
      at process.emit (node_modules/truffle/build/webpack:/node_modules/source-map-support/source-map-support.js:495:1)
      at process.emit (node_modules/truffle/build/webpack:/node_modules/source-map-support/source-map-support.js:495:1)
      at process._fatalException (internal/process/execution.js:127:25)



error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

**Comment by fainashalts** [comment]
The invalid signature error may be due to EIP-155; perhaps OVM is requiring chain id and ganache doesn't return it at this time?

**Comment by platocrat** [comment]
This issue has been fixed by #28
